build.gradle

```diff
@@ -2,7 +2,7 @@
 buildscript {
 
   ext.KOTLIN_VERSION = "1.3.61"
-  ext.ANDROID_PLUGIN_VERSION = '3.5.3'
+  ext.ANDROID_PLUGIN_VERSION = '3.6.1'
```

Comment:
I would guess this is what is causing the tests to fail

Reply:
You're right. I'm still trying to understand what's the actual change that breaks JavaFileObject when trying to get resources, and how to fix it. Any input appreciated.

    JavaFileObjects.forResource(inputFile)